lib/attributes-to-props.js

```diff
@@ -46,7 +46,7 @@ function attributesToProps(attributes) {
     }
 
     // convert inline style to object
-    if (attributes.style) {
+    if (attributes.style != null) {
```

Comment:
Just to confirm, are we not doing strict equality on purpose?

Reply:
Yes, if attributes.style is undefined or null this will return false in both cases (I think I've added corresponding unit tests). if (attributes.style) al

Reply:
Gotcha thanks for clarifying